lxd/networks_utils.go

```diff
@@ -879,6 +879,153 @@ func networkGetDnsmasqVersion() (*version.DottedVersion, error) {
 	return version.NewDottedVersion(lines[2])
 }
 
+// DHCPAllocation represents an IP allocation from dnsmasq.
```

Comment:
Any reason why this is exported? I'm sure that we have other places where we're mistakenly exporting stuff but we should probably avoid this for anything we introduce :)
`dhcpAllocation`?

Reply:
@stgraber no particular reason, I will make it unexported. :)